workflows/tn_workflow.smk

```diff
@@ -33,37 +33,37 @@ def get_insilico(wcs):
     # NOTE: sampleid and workingdir is essentially global because defined in snakefile
```

Comment:
comment no longer relevant since we arent using workingdir, right?

Reply:
The `sampleid` variable is still global, but so is anything defined in `pipeline.snakefile`. It's one of the less obvious differences between python and snakemake. As far as I understand, the `include` statement in snakemake basically just inlines the included file.